[ogre] Build failure

Package: ogre:arm64-uwp -> 14.0.1

**Host Environment**

- Host: x64-windows
- Compiler: MSVC 19.37.32822.0
-    vcpkg-tool version: 2999-12-31-unknownhash
    vcpkg-scripts version: 22fc7aad0 2023-09-06 (67 minutes ago)

**To Reproduce**

`vcpkg install `
**Failure logs**

```
-- Using cached OGRECave-ogre-v14.0.1.tar.gz.
-- Cleaning sources at C:/v/vcpkg4/buildtrees/ogre/src/v14.0.1-e1473b508d.clean. Use --editable to skip cleaning for the packages you specify.
-- Extracting source C:/v/downloads/OGRECave-ogre-v14.0.1.tar.gz
-- Applying patch fix-dependencies.patch
-- Applying patch cfg-rel-paths.patch
-- Applying patch swig-python-polyfill.patch
-- Applying patch pkgconfig.patch
-- Applying patch same-install-rules-all-platforms.patch
-- Using source at C:/v/vcpkg4/buildtrees/ogre/src/v14.0.1-e1473b508d.clean
-- Found external ninja('1.11.0').
-- Configuring arm64-uwp
-- Building arm64-uwp-dbg
CMake Error at scripts/cmake/vcpkg_execute_build_process.cmake:134 (message):
    Command failed: C:/v/downloads/tools/cmake-3.27.1-windows/cmake-3.27.1-windows-i386/bin/cmake.exe --build . --config Debug --target install -- -v -j31
    Working Directory: C:/v/vcpkg4/buildtrees/ogre/arm64-uwp-dbg
    See logs for more information:
      C:\v\vcpkg4\buildtrees\ogre\install-arm64-uwp-dbg-out.log

Call Stack (most recent call first):
  vcpkg_installed/x64-windows/share/vcpkg-cmake/vcpkg_cmake_build.cmake:74 (vcpkg_execute_build_process)
  vcpkg_installed/x64-windows/share/vcpkg-cmake/vcpkg_cmake_install.cmake:16 (vcpkg_cmake_build)
  ports/ogre/portfile.cmake:115 (vcpkg_cmake_install)
  scripts/ports.cmake:147 (include)



```
<details><summary>C:\v\vcpkg4\buildtrees\ogre\install-arm64-uwp-dbg-out.log</summary>

```
Change Dir: 'C:/v/vcpkg4/buildtrees/ogre/arm64-uwp-dbg'

Run Build Command(s): "C:/Program Files/Microsoft Visual Studio/2022/Community/Common7/IDE/CommonExtensions/Microsoft/CMake/Ninja/ninja.exe" -v -v -j31 install
[1/368] C:\PROGRA~1\MICROS~1\2022\COMMUN~1\VC\Tools\MSVC\1437~1.328\bin\Hostx64\arm64\cl.exe   /TP -DOgreMain_EXPORTS -D_MT -D_USRDLL -IC:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\include\Threading -IC:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\src -IC:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\include -IC:\v\vcpkg4\buildtrees\ogre\arm64-uwp-dbg\include -IC:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain /DWIN32 /D_WINDOWS /D_UNICODE /DUNICODE /DWINAPI_FAMILY=WINAPI_FAMILY_APP /D__WRL_NO_DEFAULT_LIB__ /nologo /Z7 /MP /GS /Gd /Gm- /W3 /WX- /Zc:wchar_t /Zc:inline /Zc:forScope /fp:precise /Oy- /EHsc  /utf-8  /MP /Zm256 /D_DEBUG /MDd /Od /RTC1  /Oi /bigobj -std:c++14   /wd4661 /wd4251 /wd4275 /showIncludes /FoOgreMain\CMakeFiles\OgreMain.dir\src\OgreControllerManager.cpp.obj /FdOgreMain\CMakeFiles\OgreMain.dir\ /FS -c C:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\src\OgreControllerManager.cpp
C:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\include\OgreWorkQueue.h(194): warning C4244: 'argument': conversion from 'Ogre::uint64' to 'unsigned long', possible loss of data
C:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\include\OgreSceneManager.h(2250): warning C4996: 'Ogre::AnimationStateSet::getAnimationStateIterator': was declared deprecated
[2/368] C:\PROGRA~1\MICROS~1\2022\COMMUN~1\VC\Tools\MSVC\1437~1.328\bin\Hostx64\arm64\cl.exe   /TP -DOgreMain_EXPORTS -D_MT -D_USRDLL -IC:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\include\Threading -IC:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\src -IC:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\include -IC:\v\vcpkg4\buildtrees\ogre\arm64-uwp-dbg\include -IC:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain /DWIN32 /D_WINDOWS /D_UNICODE /DUNICODE /DWINAPI_FAMILY=WINAPI_FAMILY_APP /D__WRL_NO_DEFAULT_LIB__ /nologo /Z7 /MP /GS /Gd /Gm- /W3 /WX- /Zc:wchar_t /Zc:inline /Zc:forScope /fp:precise /Oy- /EHsc  /utf-8  /MP /Zm256 /D_DEBUG /MDd /Od /RTC1  /Oi /bigobj -std:c++14   /wd4661 /wd4251 /wd4275 /showIncludes /FoOgreMain\CMakeFiles\OgreMain.dir\src\OgreAlignedAllocator.cpp.obj /FdOgreMain\CMakeFiles\OgreMain.dir\ /FS -c C:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\src\OgreAlignedAllocator.cpp
C:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\include\OgreWorkQueue.h(194): warning C4244: 'argument': conversion from 'Ogre::uint64' to 'unsigned long', possible loss of data
C:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\include\OgreSceneManager.h(2250): warning C4996: 'Ogre::AnimationStateSet::getAnimationStateIterator': was declared deprecated
...
Skipped 728 lines
...
[163/368] C:\PROGRA~1\MICROS~1\2022\COMMUN~1\VC\Tools\MSVC\1437~1.328\bin\Hostx64\arm64\cl.exe   /TP -DOGRED3DENGINEDLL_EXPORTS -DRenderSystem_Direct3D11_EXPORTS -D_MT -D_USRDLL -IC:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\RenderSystems\Direct3D11\include -I"C:\Program Files (x86)\Windows Kits\10\Include\10.0.22621.0\um" -IC:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\include -IC:\v\vcpkg4\buildtrees\ogre\arm64-uwp-dbg\include /DWIN32 /D_WINDOWS /D_UNICODE /DUNICODE /DWINAPI_FAMILY=WINAPI_FAMILY_APP /D__WRL_NO_DEFAULT_LIB__ /nologo /Z7 /MP /GS /Gd /Gm- /W3 /WX- /Zc:wchar_t /Zc:inline /Zc:forScope /fp:precise /Oy- /EHsc  /utf-8  /MP /Zm256 /D_DEBUG /MDd /Od /RTC1  /Oi /bigobj -std:c++14   /wd4661 /wd4251 /wd4275 /showIncludes /FoRenderSystems\Direct3D11\CMakeFiles\RenderSystem_Direct3D11.dir\src\OgreD3D11HLSLProgram.cpp.obj /FdRenderSystems\Direct3D11\CMakeFiles\RenderSystem_Direct3D11.dir\ /FS -c C:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\RenderSystems\Direct3D11\src\OgreD3D11HLSLProgram.cpp
FAILED: RenderSystems/Direct3D11/CMakeFiles/RenderSystem_Direct3D11.dir/src/OgreD3D11HLSLProgram.cpp.obj 
C:\PROGRA~1\MICROS~1\2022\COMMUN~1\VC\Tools\MSVC\1437~1.328\bin\Hostx64\arm64\cl.exe   /TP -DOGRED3DENGINEDLL_EXPORTS -DRenderSystem_Direct3D11_EXPORTS -D_MT -D_USRDLL -IC:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\RenderSystems\Direct3D11\include -I"C:\Program Files (x86)\Windows Kits\10\Include\10.0.22621.0\um" -IC:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\include -IC:\v\vcpkg4\buildtrees\ogre\arm64-uwp-dbg\include /DWIN32 /D_WINDOWS /D_UNICODE /DUNICODE /DWINAPI_FAMILY=WINAPI_FAMILY_APP /D__WRL_NO_DEFAULT_LIB__ /nologo /Z7 /MP /GS /Gd /Gm- /W3 /WX- /Zc:wchar_t /Zc:inline /Zc:forScope /fp:precise /Oy- /EHsc  /utf-8  /MP /Zm256 /D_DEBUG /MDd /Od /RTC1  /Oi /bigobj -std:c++14   /wd4661 /wd4251 /wd4275 /showIncludes /FoRenderSystems\Direct3D11\CMakeFiles\RenderSystem_Direct3D11.dir\src\OgreD3D11HLSLProgram.cpp.obj /FdRenderSystems\Direct3D11\CMakeFiles\RenderSystem_Direct3D11.dir\ /FS -c C:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\RenderSystems\Direct3D11\src\OgreD3D11HLSLProgram.cpp
C:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\include\OgreSceneManager.h(2250): warning C4996: 'Ogre::AnimationStateSet::getAnimationStateIterator': was declared deprecated
C:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\RenderSystems\Direct3D11\include\OgreD3D11RenderSystem.h(183): error C2653: 'Windows': is not a class or namespace name
C:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\RenderSystems\Direct3D11\include\OgreD3D11RenderSystem.h(183): error C3646: 'suspendingToken': unknown override specifier
C:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\RenderSystems\Direct3D11\include\OgreD3D11RenderSystem.h(183): error C2059: syntax error: ','
C:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\RenderSystems\Direct3D11\include\OgreD3D11RenderSystem.h(183): error C2238: unexpected token(s) preceding ';'
C:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\RenderSystems\Direct3D11\src\OgreD3D11HLSLProgram.cpp(564): warning C4267: '+=': conversion from 'size_t' to 'Ogre::uint32', possible loss of data
C:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\RenderSystems\Direct3D11\src\OgreD3D11HLSLProgram.cpp(565): warning C4267: '+=': conversion from 'size_t' to 'Ogre::uint32', possible loss of data
C:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\RenderSystems\Direct3D11\src\OgreD3D11HLSLProgram.cpp(566): warning C4267: '+=': conversion from 'size_t' to 'Ogre::uint32', possible loss of data
C:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\RenderSystems\Direct3D11\src\OgreD3D11HLSLProgram.cpp(567): warning C4267: '+=': conversion from 'size_t' to 'Ogre::uint32', possible loss of data
C:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\RenderSystems\Direct3D11\src\OgreD3D11HLSLProgram.cpp(568): warning C4267: '+=': conversion from 'size_t' to 'Ogre::uint32', possible loss of data
C:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\RenderSystems\Direct3D11\src\OgreD3D11HLSLProgram.cpp(569): warning C4267: '+=': conversion from 'size_t' to 'Ogre::uint32', possible loss of data
C:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\RenderSystems\Direct3D11\src\OgreD3D11HLSLProgram.cpp(570): warning C4267: '+=': conversion from 'size_t' to 'Ogre::uint32', possible loss of data
C:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\RenderSystems\Direct3D11\src\OgreD3D11HLSLProgram.cpp(571): warning C4267: '+=': conversion from 'size_t' to 'Ogre::uint32', possible loss of data
C:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\RenderSystems\Direct3D11\src\OgreD3D11HLSLProgram.cpp(572): warning C4267: '+=': conversion from 'size_t' to 'Ogre::uint32', possible loss of data
C:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\RenderSystems\Direct3D11\src\OgreD3D11HLSLProgram.cpp(573): warning C4267: '+=': conversion from 'size_t' to 'Ogre::uint32', possible loss of data
C:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\RenderSystems\Direct3D11\src\OgreD3D11HLSLProgram.cpp(575): warning C4267: 'initializing': conversion from 'size_t' to 'int', possible loss of data
C:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\RenderSystems\Direct3D11\src\OgreD3D11HLSLProgram.cpp(731): warning C4267: 'initializing': conversion from 'size_t' to 'Ogre::uint32', possible loss of data
C:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\RenderSystems\Direct3D11\src\OgreD3D11HLSLProgram.cpp(1048): warning C4267: 'initializing': conversion from 'size_t' to 'int', possible loss of data
C:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\RenderSystems\Direct3D11\src\OgreD3D11HLSLProgram.cpp(1499): warning C4267: 'argument': conversion from 'size_t' to 'UINT', possible loss of data
[164/368] C:\PROGRA~1\MICROS~1\2022\COMMUN~1\VC\Tools\MSVC\1437~1.328\bin\Hostx64\arm64\cl.exe   /TP -DOgreMain_EXPORTS -D_MT -D_USRDLL -IC:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\include\Threading -IC:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\src -IC:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\include -IC:\v\vcpkg4\buildtrees\ogre\arm64-uwp-dbg\include -IC:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain /DWIN32 /D_WINDOWS /D_UNICODE /DUNICODE /DWINAPI_FAMILY=WINAPI_FAMILY_APP /D__WRL_NO_DEFAULT_LIB__ /nologo /Z7 /MP /GS /Gd /Gm- /W3 /WX- /Zc:wchar_t /Zc:inline /Zc:forScope /fp:precise /Oy- /EHsc  /utf-8  /MP /Zm256 /D_DEBUG /MDd /Od /RTC1  /Oi /bigobj -std:c++14   /wd4661 /wd4251 /wd4275 /showIncludes /FoOgreMain\CMakeFiles\OgreMain.dir\src\OgreVector.cpp.obj /FdOgreMain\CMakeFiles\OgreMain.dir\ /FS -c C:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\src\OgreVector.cpp
C:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\include\OgreWorkQueue.h(194): warning C4244: 'argument': conversion from 'Ogre::uint64' to 'unsigned long', possible loss of data
C:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\include\OgreSceneManager.h(2250): warning C4996: 'Ogre::AnimationStateSet::getAnimationStateIterator': was declared deprecated
[165/368] C:\PROGRA~1\MICROS~1\2022\COMMUN~1\VC\Tools\MSVC\1437~1.328\bin\Hostx64\arm64\cl.exe   /TP -DOGRED3DENGINEDLL_EXPORTS -DRenderSystem_Direct3D11_EXPORTS -D_MT -D_USRDLL -IC:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\RenderSystems\Direct3D11\include -I"C:\Program Files (x86)\Windows Kits\10\Include\10.0.22621.0\um" -IC:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\include -IC:\v\vcpkg4\buildtrees\ogre\arm64-uwp-dbg\include /DWIN32 /D_WINDOWS /D_UNICODE /DUNICODE /DWINAPI_FAMILY=WINAPI_FAMILY_APP /D__WRL_NO_DEFAULT_LIB__ /nologo /Z7 /MP /GS /Gd /Gm- /W3 /WX- /Zc:wchar_t /Zc:inline /Zc:forScope /fp:precise /Oy- /EHsc  /utf-8  /MP /Zm256 /D_DEBUG /MDd /Od /RTC1  /Oi /bigobj -std:c++14   /wd4661 /wd4251 /wd4275 /showIncludes /FoRenderSystems\Direct3D11\CMakeFiles\RenderSystem_Direct3D11.dir\src\OgreD3D11HardwareBuffer.cpp.obj /FdRenderSystems\Direct3D11\CMakeFiles\RenderSystem_Direct3D11.dir\ /FS -c C:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\RenderSystems\Direct3D11\src\OgreD3D11HardwareBuffer.cpp
C:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\RenderSystems\Direct3D11\src\OgreD3D11HardwareBuffer.cpp(205): warning C4267: '=': conversion from 'size_t' to 'UINT', possible loss of data
[166/368] C:\PROGRA~1\MICROS~1\2022\COMMUN~1\VC\Tools\MSVC\1437~1.328\bin\Hostx64\arm64\cl.exe   /TP -DOGRED3DENGINEDLL_EXPORTS -DRenderSystem_Direct3D11_EXPORTS -D_MT -D_USRDLL -IC:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\RenderSystems\Direct3D11\include -I"C:\Program Files (x86)\Windows Kits\10\Include\10.0.22621.0\um" -IC:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\include -IC:\v\vcpkg4\buildtrees\ogre\arm64-uwp-dbg\include /DWIN32 /D_WINDOWS /D_UNICODE /DUNICODE /DWINAPI_FAMILY=WINAPI_FAMILY_APP /D__WRL_NO_DEFAULT_LIB__ /nologo /Z7 /MP /GS /Gd /Gm- /W3 /WX- /Zc:wchar_t /Zc:inline /Zc:forScope /fp:precise /Oy- /EHsc  /utf-8  /MP /Zm256 /D_DEBUG /MDd /Od /RTC1  /Oi /bigobj -std:c++14   /wd4661 /wd4251 /wd4275 /showIncludes /FoRenderSystems\Direct3D11\CMakeFiles\RenderSystem_Direct3D11.dir\src\OgreD3D11EngineDll.cpp.obj /FdRenderSystems\Direct3D11\CMakeFiles\RenderSystem_Direct3D11.dir\ /FS -c C:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\RenderSystems\Direct3D11\src\OgreD3D11EngineDll.cpp
FAILED: RenderSystems/Direct3D11/CMakeFiles/RenderSystem_Direct3D11.dir/src/OgreD3D11EngineDll.cpp.obj 
C:\PROGRA~1\MICROS~1\2022\COMMUN~1\VC\Tools\MSVC\1437~1.328\bin\Hostx64\arm64\cl.exe   /TP -DOGRED3DENGINEDLL_EXPORTS -DRenderSystem_Direct3D11_EXPORTS -D_MT -D_USRDLL -IC:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\RenderSystems\Direct3D11\include -I"C:\Program Files (x86)\Windows Kits\10\Include\10.0.22621.0\um" -IC:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\include -IC:\v\vcpkg4\buildtrees\ogre\arm64-uwp-dbg\include /DWIN32 /D_WINDOWS /D_UNICODE /DUNICODE /DWINAPI_FAMILY=WINAPI_FAMILY_APP /D__WRL_NO_DEFAULT_LIB__ /nologo /Z7 /MP /GS /Gd /Gm- /W3 /WX- /Zc:wchar_t /Zc:inline /Zc:forScope /fp:precise /Oy- /EHsc  /utf-8  /MP /Zm256 /D_DEBUG /MDd /Od /RTC1  /Oi /bigobj -std:c++14   /wd4661 /wd4251 /wd4275 /showIncludes /FoRenderSystems\Direct3D11\CMakeFiles\RenderSystem_Direct3D11.dir\src\OgreD3D11EngineDll.cpp.obj /FdRenderSystems\Direct3D11\CMakeFiles\RenderSystem_Direct3D11.dir\ /FS -c C:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\RenderSystems\Direct3D11\src\OgreD3D11EngineDll.cpp
C:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\include\OgreSceneManager.h(2250): warning C4996: 'Ogre::AnimationStateSet::getAnimationStateIterator': was declared deprecated
C:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\RenderSystems\Direct3D11\include\OgreD3D11RenderSystem.h(183): error C2653: 'Windows': is not a class or namespace name
C:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\RenderSystems\Direct3D11\include\OgreD3D11RenderSystem.h(183): error C3646: 'suspendingToken': unknown override specifier
C:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\RenderSystems\Direct3D11\include\OgreD3D11RenderSystem.h(183): error C2059: syntax error: ','
C:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\RenderSystems\Direct3D11\include\OgreD3D11RenderSystem.h(183): error C2238: unexpected token(s) preceding ';'
[167/368] C:\PROGRA~1\MICROS~1\2022\COMMUN~1\VC\Tools\MSVC\1437~1.328\bin\Hostx64\arm64\cl.exe   /TP -DOGRED3DENGINEDLL_EXPORTS -DRenderSystem_Direct3D11_EXPORTS -D_MT -D_USRDLL -IC:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\RenderSystems\Direct3D11\include -I"C:\Program Files (x86)\Windows Kits\10\Include\10.0.22621.0\um" -IC:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\include -IC:\v\vcpkg4\buildtrees\ogre\arm64-uwp-dbg\include /DWIN32 /D_WINDOWS /D_UNICODE /DUNICODE /DWINAPI_FAMILY=WINAPI_FAMILY_APP /D__WRL_NO_DEFAULT_LIB__ /nologo /Z7 /MP /GS /Gd /Gm- /W3 /WX- /Zc:wchar_t /Zc:inline /Zc:forScope /fp:precise /Oy- /EHsc  /utf-8  /MP /Zm256 /D_DEBUG /MDd /Od /RTC1  /Oi /bigobj -std:c++14   /wd4661 /wd4251 /wd4275 /showIncludes /FoRenderSystems\Direct3D11\CMakeFiles\RenderSystem_Direct3D11.dir\src\OgreD3D11HLSLProgramFactory.cpp.obj /FdRenderSystems\Direct3D11\CMakeFiles\RenderSystem_Direct3D11.dir\ /FS -c C:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\RenderSystems\Direct3D11\src\OgreD3D11HLSLProgramFactory.cpp
[168/368] C:\PROGRA~1\MICROS~1\2022\COMMUN~1\VC\Tools\MSVC\1437~1.328\bin\Hostx64\arm64\cl.exe   /TP -DOgreMain_EXPORTS -D_MT -D_USRDLL -IC:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\include\Threading -IC:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\src -IC:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\include -IC:\v\vcpkg4\buildtrees\ogre\arm64-uwp-dbg\include -IC:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain /DWIN32 /D_WINDOWS /D_UNICODE /DUNICODE /DWINAPI_FAMILY=WINAPI_FAMILY_APP /D__WRL_NO_DEFAULT_LIB__ /nologo /Z7 /MP /GS /Gd /Gm- /W3 /WX- /Zc:wchar_t /Zc:inline /Zc:forScope /fp:precise /Oy- /EHsc  /utf-8  /MP /Zm256 /D_DEBUG /MDd /Od /RTC1  /Oi /bigobj -std:c++14   /wd4661 /wd4251 /wd4275 /showIncludes /FoOgreMain\CMakeFiles\OgreMain.dir\src\OgreWorkQueue.cpp.obj /FdOgreMain\CMakeFiles\OgreMain.dir\ /FS -c C:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\src\OgreWorkQueue.cpp
C:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\include\OgreWorkQueue.h(194): warning C4244: 'argument': conversion from 'Ogre::uint64' to 'unsigned long', possible loss of data
C:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\include\OgreSceneManager.h(2250): warning C4996: 'Ogre::AnimationStateSet::getAnimationStateIterator': was declared deprecated
C:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\src\OgreWorkQueue.cpp(172): warning C4244: 'initializing': conversion from 'uint64_t' to 'unsigned long', possible loss of data
C:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\src\OgreWorkQueue.cpp(194): warning C4244: '=': conversion from 'uint64_t' to 'unsigned long', possible loss of data
[169/368] C:\PROGRA~1\MICROS~1\2022\COMMUN~1\VC\Tools\MSVC\1437~1.328\bin\Hostx64\arm64\cl.exe   /TP -DOgreMain_EXPORTS -D_MT -D_USRDLL -IC:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\include\Threading -IC:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\src -IC:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\include -IC:\v\vcpkg4\buildtrees\ogre\arm64-uwp-dbg\include -IC:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain /DWIN32 /D_WINDOWS /D_UNICODE /DUNICODE /DWINAPI_FAMILY=WINAPI_FAMILY_APP /D__WRL_NO_DEFAULT_LIB__ /nologo /Z7 /MP /GS /Gd /Gm- /W3 /WX- /Zc:wchar_t /Zc:inline /Zc:forScope /fp:precise /Oy- /EHsc  /utf-8  /MP /Zm256 /D_DEBUG /MDd /Od /RTC1  /Oi /bigobj -std:c++14   /wd4661 /wd4251 /wd4275 /showIncludes /FoOgreMain\CMakeFiles\OgreMain.dir\src\OgreVertexIndexData.cpp.obj /FdOgreMain\CMakeFiles\OgreMain.dir\ /FS -c C:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\src\OgreVertexIndexData.cpp
C:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\include\OgreWorkQueue.h(194): warning C4244: 'argument': conversion from 'Ogre::uint64' to 'unsigned long', possible loss of data
C:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\include\OgreSceneManager.h(2250): warning C4996: 'Ogre::AnimationStateSet::getAnimationStateIterator': was declared deprecated
C:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\src\OgreVertexIndexData.cpp(99): warning C4267: 'argument': conversion from 'size_t' to 'int', possible loss of data
C:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\src\OgreVertexIndexData.cpp(266): warning C4267: 'argument': conversion from 'size_t' to 'Ogre::uint32', possible loss of data
C:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\src\OgreVertexIndexData.cpp(270): warning C4267: 'argument': conversion from 'size_t' to 'Ogre::uint32', possible loss of data
C:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\src\OgreVertexIndexData.cpp(276): warning C4267: 'argument': conversion from 'size_t' to 'Ogre::uint32', possible loss of data
C:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\src\OgreVertexIndexData.cpp(281): warning C4267: 'argument': conversion from 'size_t' to 'Ogre::uint32', possible loss of data
C:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\src\OgreVertexIndexData.cpp(343): warning C4267: 'argument': conversion from 'size_t' to 'Ogre::uint32', possible loss of data
C:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\src\OgreVertexIndexData.cpp(69): warning C4244: '=': conversion from 'int32_t' to 'float', possible loss of data
C:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\src\OgreVertexIndexData.cpp(276): note: see reference to function template instantiation 'void Ogre::unpack_10_10_10_2<false>(Ogre::uint8 *,Ogre::uint8 *,int)' being compiled
[170/368] C:\PROGRA~1\MICROS~1\2022\COMMUN~1\VC\Tools\MSVC\1437~1.328\bin\Hostx64\arm64\cl.exe   /TP -DOgreMain_EXPORTS -D_MT -D_USRDLL -IC:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\include\Threading -IC:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\src -IC:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\include -IC:\v\vcpkg4\buildtrees\ogre\arm64-uwp-dbg\include -IC:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain /DWIN32 /D_WINDOWS /D_UNICODE /DUNICODE /DWINAPI_FAMILY=WINAPI_FAMILY_APP /D__WRL_NO_DEFAULT_LIB__ /nologo /Z7 /MP /GS /Gd /Gm- /W3 /WX- /Zc:wchar_t /Zc:inline /Zc:forScope /fp:precise /Oy- /EHsc  /utf-8  /MP /Zm256 /D_DEBUG /MDd /Od /RTC1  /Oi /bigobj -std:c++14   /wd4661 /wd4251 /wd4275 /showIncludes /FoOgreMain\CMakeFiles\OgreMain.dir\src\OgreTexture.cpp.obj /FdOgreMain\CMakeFiles\OgreMain.dir\ /FS -c C:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\src\OgreTexture.cpp
C:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\include\OgreWorkQueue.h(194): warning C4244: 'argument': conversion from 'Ogre::uint64' to 'unsigned long', possible loss of data
C:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\include\OgreSceneManager.h(2250): warning C4996: 'Ogre::AnimationStateSet::getAnimationStateIterator': was declared deprecated
C:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\src\OgreTexture.cpp(130): warning C4244: '=': conversion from 'Ogre::ushort' to 'Ogre::uchar', possible loss of data
C:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\src\OgreTexture.cpp(140): warning C4244: '=': conversion from 'Ogre::ushort' to 'Ogre::uchar', possible loss of data
C:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\src\OgreTexture.cpp(150): warning C4244: '=': conversion from 'Ogre::ushort' to 'Ogre::uchar', possible loss of data
C:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\src\OgreTexture.cpp(151): warning C4244: '=': conversion from 'Ogre::ushort' to 'Ogre::uchar', possible loss of data
[171/368] C:\PROGRA~1\MICROS~1\2022\COMMUN~1\VC\Tools\MSVC\1437~1.328\bin\Hostx64\arm64\cl.exe   /TP -DOgreMain_EXPORTS -D_MT -D_USRDLL -IC:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\include\Threading -IC:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\src -IC:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\include -IC:\v\vcpkg4\buildtrees\ogre\arm64-uwp-dbg\include -IC:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain /DWIN32 /D_WINDOWS /D_UNICODE /DUNICODE /DWINAPI_FAMILY=WINAPI_FAMILY_APP /D__WRL_NO_DEFAULT_LIB__ /nologo /Z7 /MP /GS /Gd /Gm- /W3 /WX- /Zc:wchar_t /Zc:inline /Zc:forScope /fp:precise /Oy- /EHsc  /utf-8  /MP /Zm256 /D_DEBUG /MDd /Od /RTC1  /Oi /bigobj -std:c++14   /wd4661 /wd4251 /wd4275 /showIncludes /FoOgreMain\CMakeFiles\OgreMain.dir\src\OgreSkyRenderer.cpp.obj /FdOgreMain\CMakeFiles\OgreMain.dir\ /FS -c C:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\src\OgreSkyRenderer.cpp
C:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\include\OgreWorkQueue.h(194): warning C4244: 'argument': conversion from 'Ogre::uint64' to 'unsigned long', possible loss of data
C:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\include\OgreSceneManager.h(2250): warning C4996: 'Ogre::AnimationStateSet::getAnimationStateIterator': was declared deprecated
[172/368] C:\PROGRA~1\MICROS~1\2022\COMMUN~1\VC\Tools\MSVC\1437~1.328\bin\Hostx64\arm64\cl.exe   /TP -DOGRED3DENGINEDLL_EXPORTS -DRenderSystem_Direct3D11_EXPORTS -D_MT -D_USRDLL -IC:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\RenderSystems\Direct3D11\include -I"C:\Program Files (x86)\Windows Kits\10\Include\10.0.22621.0\um" -IC:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\include -IC:\v\vcpkg4\buildtrees\ogre\arm64-uwp-dbg\include /DWIN32 /D_WINDOWS /D_UNICODE /DUNICODE /DWINAPI_FAMILY=WINAPI_FAMILY_APP /D__WRL_NO_DEFAULT_LIB__ /nologo /Z7 /MP /GS /Gd /Gm- /W3 /WX- /Zc:wchar_t /Zc:inline /Zc:forScope /fp:precise /Oy- /EHsc  /utf-8  /MP /Zm256 /D_DEBUG /MDd /Od /RTC1  /Oi /bigobj -std:c++14   /wd4661 /wd4251 /wd4275 /showIncludes /FoRenderSystems\Direct3D11\CMakeFiles\RenderSystem_Direct3D11.dir\src\OgreD3D11DeviceResource.cpp.obj /FdRenderSystems\Direct3D11\CMakeFiles\RenderSystem_Direct3D11.dir\ /FS -c C:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\RenderSystems\Direct3D11\src\OgreD3D11DeviceResource.cpp
[173/368] C:\PROGRA~1\MICROS~1\2022\COMMUN~1\VC\Tools\MSVC\1437~1.328\bin\Hostx64\arm64\cl.exe   /TP -DOgreMain_EXPORTS -D_MT -D_USRDLL -IC:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\include\Threading -IC:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\src -IC:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\include -IC:\v\vcpkg4\buildtrees\ogre\arm64-uwp-dbg\include -IC:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain /DWIN32 /D_WINDOWS /D_UNICODE /DUNICODE /DWINAPI_FAMILY=WINAPI_FAMILY_APP /D__WRL_NO_DEFAULT_LIB__ /nologo /Z7 /MP /GS /Gd /Gm- /W3 /WX- /Zc:wchar_t /Zc:inline /Zc:forScope /fp:precise /Oy- /EHsc  /utf-8  /MP /Zm256 /D_DEBUG /MDd /Od /RTC1  /Oi /bigobj -std:c++14   /wd4661 /wd4251 /wd4275 /showIncludes /FoOgreMain\CMakeFiles\OgreMain.dir\src\OgreTextureManager.cpp.obj /FdOgreMain\CMakeFiles\OgreMain.dir\ /FS -c C:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\src\OgreTextureManager.cpp
C:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\include\OgreWorkQueue.h(194): warning C4244: 'argument': conversion from 'Ogre::uint64' to 'unsigned long', possible loss of data
C:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\include\OgreSceneManager.h(2250): warning C4996: 'Ogre::AnimationStateSet::getAnimationStateIterator': was declared deprecated
[174/368] C:\PROGRA~1\MICROS~1\2022\COMMUN~1\VC\Tools\MSVC\1437~1.328\bin\Hostx64\arm64\cl.exe   /TP -DOgreMain_EXPORTS -D_MT -D_USRDLL -IC:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\include\Threading -IC:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\src -IC:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\include -IC:\v\vcpkg4\buildtrees\ogre\arm64-uwp-dbg\include -IC:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain /DWIN32 /D_WINDOWS /D_UNICODE /DUNICODE /DWINAPI_FAMILY=WINAPI_FAMILY_APP /D__WRL_NO_DEFAULT_LIB__ /nologo /Z7 /MP /GS /Gd /Gm- /W3 /WX- /Zc:wchar_t /Zc:inline /Zc:forScope /fp:precise /Oy- /EHsc  /utf-8  /MP /Zm256 /D_DEBUG /MDd /Od /RTC1  /Oi /bigobj -std:c++14   /wd4661 /wd4251 /wd4275 /showIncludes /FoOgreMain\CMakeFiles\OgreMain.dir\src\OgreDDSCodec.cpp.obj /FdOgreMain\CMakeFiles\OgreMain.dir\ /FS -c C:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\src\OgreDDSCodec.cpp
C:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\include\OgreWorkQueue.h(194): warning C4244: 'argument': conversion from 'Ogre::uint64' to 'unsigned long', possible loss of data
C:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\include\OgreSceneManager.h(2250): warning C4996: 'Ogre::AnimationStateSet::getAnimationStateIterator': was declared deprecated
C:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\src\OgreDDSCodec.cpp(396): warning C4267: 'argument': conversion from 'size_t' to 'Ogre::uint32', possible loss of data
C:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\src\OgreDDSCodec.cpp(398): warning C4267: 'argument': conversion from 'size_t' to 'Ogre::uint32', possible loss of data
[175/368] C:\PROGRA~1\MICROS~1\2022\COMMUN~1\VC\Tools\MSVC\1437~1.328\bin\Hostx64\arm64\cl.exe   /TP -DOgreMain_EXPORTS -D_MT -D_USRDLL -IC:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\include\Threading -IC:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\src -IC:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\include -IC:\v\vcpkg4\buildtrees\ogre\arm64-uwp-dbg\include -IC:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain /DWIN32 /D_WINDOWS /D_UNICODE /DUNICODE /DWINAPI_FAMILY=WINAPI_FAMILY_APP /D__WRL_NO_DEFAULT_LIB__ /nologo /Z7 /MP /GS /Gd /Gm- /W3 /WX- /Zc:wchar_t /Zc:inline /Zc:forScope /fp:precise /Oy- /EHsc  /utf-8  /MP /Zm256 /D_DEBUG /MDd /Od /RTC1  /Oi /bigobj -std:c++14   /wd4661 /wd4251 /wd4275 /showIncludes /FoOgreMain\CMakeFiles\OgreMain.dir\src\OgreStringInterface.cpp.obj /FdOgreMain\CMakeFiles\OgreMain.dir\ /FS -c C:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\src\OgreStringInterface.cpp
C:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\include\OgreWorkQueue.h(194): warning C4244: 'argument': conversion from 'Ogre::uint64' to 'unsigned long', possible loss of data
C:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\include\OgreSceneManager.h(2250): warning C4996: 'Ogre::AnimationStateSet::getAnimationStateIterator': was declared deprecated
[176/368] C:\PROGRA~1\MICROS~1\2022\COMMUN~1\VC\Tools\MSVC\1437~1.328\bin\Hostx64\arm64\cl.exe   /TP -DOgreMain_EXPORTS -D_MT -D_USRDLL -IC:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\include\Threading -IC:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\src -IC:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\include -IC:\v\vcpkg4\buildtrees\ogre\arm64-uwp-dbg\include -IC:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain /DWIN32 /D_WINDOWS /D_UNICODE /DUNICODE /DWINAPI_FAMILY=WINAPI_FAMILY_APP /D__WRL_NO_DEFAULT_LIB__ /nologo /Z7 /MP /GS /Gd /Gm- /W3 /WX- /Zc:wchar_t /Zc:inline /Zc:forScope /fp:precise /Oy- /EHsc  /utf-8  /MP /Zm256 /D_DEBUG /MDd /Od /RTC1  /Oi /bigobj -std:c++14   /wd4661 /wd4251 /wd4275 /showIncludes /FoOgreMain\CMakeFiles\OgreMain.dir\src\OgreWireBoundingBox.cpp.obj /FdOgreMain\CMakeFiles\OgreMain.dir\ /FS -c C:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\src\OgreWireBoundingBox.cpp
C:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\include\OgreWorkQueue.h(194): warning C4244: 'argument': conversion from 'Ogre::uint64' to 'unsigned long', possible loss of data
C:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\include\OgreSceneManager.h(2250): warning C4996: 'Ogre::AnimationStateSet::getAnimationStateIterator': was declared deprecated
[177/368] C:\PROGRA~1\MICROS~1\2022\COMMUN~1\VC\Tools\MSVC\1437~1.328\bin\Hostx64\arm64\cl.exe   /TP -DOgreMain_EXPORTS -D_MT -D_USRDLL -IC:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\include\Threading -IC:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\src -IC:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\include -IC:\v\vcpkg4\buildtrees\ogre\arm64-uwp-dbg\include -IC:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain /DWIN32 /D_WINDOWS /D_UNICODE /DUNICODE /DWINAPI_FAMILY=WINAPI_FAMILY_APP /D__WRL_NO_DEFAULT_LIB__ /nologo /Z7 /MP /GS /Gd /Gm- /W3 /WX- /Zc:wchar_t /Zc:inline /Zc:forScope /fp:precise /Oy- /EHsc  /utf-8  /MP /Zm256 /D_DEBUG /MDd /Od /RTC1  /Oi /bigobj -std:c++14   /wd4661 /wd4251 /wd4275 /showIncludes /FoOgreMain\CMakeFiles\OgreMain.dir\src\OgreResource.cpp.obj /FdOgreMain\CMakeFiles\OgreMain.dir\ /FS -c C:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\src\OgreResource.cpp
C:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\include\OgreWorkQueue.h(194): warning C4244: 'argument': conversion from 'Ogre::uint64' to 'unsigned long', possible loss of data
C:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\include\OgreSceneManager.h(2250): warning C4996: 'Ogre::AnimationStateSet::getAnimationStateIterator': was declared deprecated
[178/368] C:\PROGRA~1\MICROS~1\2022\COMMUN~1\VC\Tools\MSVC\1437~1.328\bin\Hostx64\arm64\cl.exe   /TP -DOgreMain_EXPORTS -D_MT -D_USRDLL -IC:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\include\Threading -IC:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\src -IC:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\include -IC:\v\vcpkg4\buildtrees\ogre\arm64-uwp-dbg\include -IC:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain /DWIN32 /D_WINDOWS /D_UNICODE /DUNICODE /DWINAPI_FAMILY=WINAPI_FAMILY_APP /D__WRL_NO_DEFAULT_LIB__ /nologo /Z7 /MP /GS /Gd /Gm- /W3 /WX- /Zc:wchar_t /Zc:inline /Zc:forScope /fp:precise /Oy- /EHsc  /utf-8  /MP /Zm256 /D_DEBUG /MDd /Od /RTC1  /Oi /bigobj -std:c++14   /wd4661 /wd4251 /wd4275 /showIncludes /FoOgreMain\CMakeFiles\OgreMain.dir\src\OgreETCCodec.cpp.obj /FdOgreMain\CMakeFiles\OgreMain.dir\ /FS -c C:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\src\OgreETCCodec.cpp
C:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\include\OgreWorkQueue.h(194): warning C4244: 'argument': conversion from 'Ogre::uint64' to 'unsigned long', possible loss of data
C:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\include\OgreSceneManager.h(2250): warning C4996: 'Ogre::AnimationStateSet::getAnimationStateIterator': was declared deprecated
[179/368] C:\PROGRA~1\MICROS~1\2022\COMMUN~1\VC\Tools\MSVC\1437~1.328\bin\Hostx64\arm64\cl.exe   /TP -DOgreMain_EXPORTS -D_MT -D_USRDLL -IC:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\include\Threading -IC:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\src -IC:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\include -IC:\v\vcpkg4\buildtrees\ogre\arm64-uwp-dbg\include -IC:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain /DWIN32 /D_WINDOWS /D_UNICODE /DUNICODE /DWINAPI_FAMILY=WINAPI_FAMILY_APP /D__WRL_NO_DEFAULT_LIB__ /nologo /Z7 /MP /GS /Gd /Gm- /W3 /WX- /Zc:wchar_t /Zc:inline /Zc:forScope /fp:precise /Oy- /EHsc  /utf-8  /MP /Zm256 /D_DEBUG /MDd /Od /RTC1  /Oi /bigobj -std:c++14   /wd4661 /wd4251 /wd4275 /showIncludes /FoOgreMain\CMakeFiles\OgreMain.dir\src\OgreUserObjectBindings.cpp.obj /FdOgreMain\CMakeFiles\OgreMain.dir\ /FS -c C:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\src\OgreUserObjectBindings.cpp
C:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\include\OgreWorkQueue.h(194): warning C4244: 'argument': conversion from 'Ogre::uint64' to 'unsigned long', possible loss of data
C:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\include\OgreSceneManager.h(2250): warning C4996: 'Ogre::AnimationStateSet::getAnimationStateIterator': was declared deprecated
[180/368] C:\PROGRA~1\MICROS~1\2022\COMMUN~1\VC\Tools\MSVC\1437~1.328\bin\Hostx64\arm64\cl.exe   /TP -DOgreMain_EXPORTS -D_MT -D_USRDLL -IC:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\include\Threading -IC:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\src -IC:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\include -IC:\v\vcpkg4\buildtrees\ogre\arm64-uwp-dbg\include -IC:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain /DWIN32 /D_WINDOWS /D_UNICODE /DUNICODE /DWINAPI_FAMILY=WINAPI_FAMILY_APP /D__WRL_NO_DEFAULT_LIB__ /nologo /Z7 /MP /GS /Gd /Gm- /W3 /WX- /Zc:wchar_t /Zc:inline /Zc:forScope /fp:precise /Oy- /EHsc  /utf-8  /MP /Zm256 /D_DEBUG /MDd /Od /RTC1  /Oi /bigobj -std:c++14   /wd4661 /wd4251 /wd4275 /showIncludes /FoOgreMain\CMakeFiles\OgreMain.dir\src\OgreScriptCompiler.cpp.obj /FdOgreMain\CMakeFiles\OgreMain.dir\ /FS -c C:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\src\OgreScriptCompiler.cpp
C:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\include\OgreWorkQueue.h(194): warning C4244: 'argument': conversion from 'Ogre::uint64' to 'unsigned long', possible loss of data
C:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\include\OgreSceneManager.h(2250): warning C4996: 'Ogre::AnimationStateSet::getAnimationStateIterator': was declared deprecated
[181/368] C:\PROGRA~1\MICROS~1\2022\COMMUN~1\VC\Tools\MSVC\1437~1.328\bin\Hostx64\arm64\cl.exe   /TP -DOgreMain_EXPORTS -D_MT -D_USRDLL -IC:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\include\Threading -IC:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\src -IC:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\include -IC:\v\vcpkg4\buildtrees\ogre\arm64-uwp-dbg\include -IC:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain /DWIN32 /D_WINDOWS /D_UNICODE /DUNICODE /DWINAPI_FAMILY=WINAPI_FAMILY_APP /D__WRL_NO_DEFAULT_LIB__ /nologo /Z7 /MP /GS /Gd /Gm- /W3 /WX- /Zc:wchar_t /Zc:inline /Zc:forScope /fp:precise /Oy- /EHsc  /utf-8  /MP /Zm256 /D_DEBUG /MDd /Od /RTC1  /Oi /bigobj -std:c++14   /wd4661 /wd4251 /wd4275 /showIncludes /FoOgreMain\CMakeFiles\OgreMain.dir\src\OgreShadowVolumeExtrudeProgram.cpp.obj /FdOgreMain\CMakeFiles\OgreMain.dir\ /FS -c C:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\src\OgreShadowVolumeExtrudeProgram.cpp
C:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\include\OgreWorkQueue.h(194): warning C4244: 'argument': conversion from 'Ogre::uint64' to 'unsigned long', possible loss of data
C:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\include\OgreSceneManager.h(2250): warning C4996: 'Ogre::AnimationStateSet::getAnimationStateIterator': was declared deprecated
[182/368] C:\PROGRA~1\MICROS~1\2022\COMMUN~1\VC\Tools\MSVC\1437~1.328\bin\Hostx64\arm64\cl.exe   /TP -DOgreMain_EXPORTS -D_MT -D_USRDLL -IC:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\include\Threading -IC:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\src -IC:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\include -IC:\v\vcpkg4\buildtrees\ogre\arm64-uwp-dbg\include -IC:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain /DWIN32 /D_WINDOWS /D_UNICODE /DUNICODE /DWINAPI_FAMILY=WINAPI_FAMILY_APP /D__WRL_NO_DEFAULT_LIB__ /nologo /Z7 /MP /GS /Gd /Gm- /W3 /WX- /Zc:wchar_t /Zc:inline /Zc:forScope /fp:precise /Oy- /EHsc  /utf-8  /MP /Zm256 /D_DEBUG /MDd /Od /RTC1  /Oi /bigobj -std:c++14   /wd4661 /wd4251 /wd4275 /showIncludes /FoOgreMain\CMakeFiles\OgreMain.dir\src\OgreSimpleSpline.cpp.obj /FdOgreMain\CMakeFiles\OgreMain.dir\ /FS -c C:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\src\OgreSimpleSpline.cpp
C:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\include\OgreWorkQueue.h(194): warning C4244: 'argument': conversion from 'Ogre::uint64' to 'unsigned long', possible loss of data
C:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\include\OgreSceneManager.h(2250): warning C4996: 'Ogre::AnimationStateSet::getAnimationStateIterator': was declared deprecated
[183/368] C:\PROGRA~1\MICROS~1\2022\COMMUN~1\VC\Tools\MSVC\1437~1.328\bin\Hostx64\arm64\cl.exe   /TP -DOgreMain_EXPORTS -D_MT -D_USRDLL -IC:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\include\Threading -IC:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\src -IC:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\include -IC:\v\vcpkg4\buildtrees\ogre\arm64-uwp-dbg\include -IC:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain /DWIN32 /D_WINDOWS /D_UNICODE /DUNICODE /DWINAPI_FAMILY=WINAPI_FAMILY_APP /D__WRL_NO_DEFAULT_LIB__ /nologo /Z7 /MP /GS /Gd /Gm- /W3 /WX- /Zc:wchar_t /Zc:inline /Zc:forScope /fp:precise /Oy- /EHsc  /utf-8  /MP /Zm256 /D_DEBUG /MDd /Od /RTC1  /Oi /bigobj -std:c++14   /wd4661 /wd4251 /wd4275 /showIncludes /FoOgreMain\CMakeFiles\OgreMain.dir\src\OgreUnifiedHighLevelGpuProgram.cpp.obj /FdOgreMain\CMakeFiles\OgreMain.dir\ /FS -c C:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\src\OgreUnifiedHighLevelGpuProgram.cpp
C:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\include\OgreWorkQueue.h(194): warning C4244: 'argument': conversion from 'Ogre::uint64' to 'unsigned long', possible loss of data
C:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\include\OgreSceneManager.h(2250): warning C4996: 'Ogre::AnimationStateSet::getAnimationStateIterator': was declared deprecated
[184/368] C:\PROGRA~1\MICROS~1\2022\COMMUN~1\VC\Tools\MSVC\1437~1.328\bin\Hostx64\arm64\cl.exe   /TP -DOgreMain_EXPORTS -D_MT -D_USRDLL -IC:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\include\Threading -IC:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\src -IC:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\include -IC:\v\vcpkg4\buildtrees\ogre\arm64-uwp-dbg\include -IC:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain /DWIN32 /D_WINDOWS /D_UNICODE /DUNICODE /DWINAPI_FAMILY=WINAPI_FAMILY_APP /D__WRL_NO_DEFAULT_LIB__ /nologo /Z7 /MP /GS /Gd /Gm- /W3 /WX- /Zc:wchar_t /Zc:inline /Zc:forScope /fp:precise /Oy- /EHsc  /utf-8  /MP /Zm256 /D_DEBUG /MDd /Od /RTC1  /Oi /bigobj -std:c++14   /wd4661 /wd4251 /wd4275 /showIncludes /FoOgreMain\CMakeFiles\OgreMain.dir\src\OgreTextureUnitState.cpp.obj /FdOgreMain\CMakeFiles\OgreMain.dir\ /FS -c C:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\src\OgreTextureUnitState.cpp
C:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\include\OgreWorkQueue.h(194): warning C4244: 'argument': conversion from 'Ogre::uint64' to 'unsigned long', possible loss of data
C:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\include\OgreSceneManager.h(2250): warning C4996: 'Ogre::AnimationStateSet::getAnimationStateIterator': was declared deprecated
[185/368] C:\PROGRA~1\MICROS~1\2022\COMMUN~1\VC\Tools\MSVC\1437~1.328\bin\Hostx64\arm64\cl.exe   /TP -DOgreMain_EXPORTS -D_MT -D_USRDLL -IC:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\include\Threading -IC:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\src -IC:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\include -IC:\v\vcpkg4\buildtrees\ogre\arm64-uwp-dbg\include -IC:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain /DWIN32 /D_WINDOWS /D_UNICODE /DUNICODE /DWINAPI_FAMILY=WINAPI_FAMILY_APP /D__WRL_NO_DEFAULT_LIB__ /nologo /Z7 /MP /GS /Gd /Gm- /W3 /WX- /Zc:wchar_t /Zc:inline /Zc:forScope /fp:precise /Oy- /EHsc  /utf-8  /MP /Zm256 /D_DEBUG /MDd /Od /RTC1  /Oi /bigobj -std:c++14   /wd4661 /wd4251 /wd4275 /showIncludes /FoOgreMain\CMakeFiles\OgreMain.dir\src\OgreViewport.cpp.obj /FdOgreMain\CMakeFiles\OgreMain.dir\ /FS -c C:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\src\OgreViewport.cpp
C:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\include\OgreWorkQueue.h(194): warning C4244: 'argument': conversion from 'Ogre::uint64' to 'unsigned long', possible loss of data
C:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\include\OgreSceneManager.h(2250): warning C4996: 'Ogre::AnimationStateSet::getAnimationStateIterator': was declared deprecated
C:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\src\OgreViewport.cpp(103): warning C4244: 'argument': conversion from 'T' to 'const T', possible loss of data
        with
        [
            T=float
        ]
        and
        [
            T=Ogre::int32
        ]
C:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\src\OgreViewport.cpp(103): warning C4244: 'argument': conversion from 'T' to 'const T', possible loss of data
        with
        [
            T=float
        ]
        and
        [
            T=Ogre::int32
        ]
C:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\src\OgreViewport.cpp(103): warning C4244: 'argument': conversion from 'T' to 'const T', possible loss of data
        with
        [
            T=float
        ]
        and
        [
            T=Ogre::int32
        ]
C:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\src\OgreViewport.cpp(103): warning C4244: 'argument': conversion from 'T' to 'const T', possible loss of data
        with
        [
            T=float
        ]
        and
        [
            T=Ogre::int32
        ]
[186/368] C:\PROGRA~1\MICROS~1\2022\COMMUN~1\VC\Tools\MSVC\1437~1.328\bin\Hostx64\arm64\cl.exe   /TP -DOgreMain_EXPORTS -D_MT -D_USRDLL -IC:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\include\Threading -IC:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\src -IC:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\include -IC:\v\vcpkg4\buildtrees\ogre\arm64-uwp-dbg\include -IC:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain /DWIN32 /D_WINDOWS /D_UNICODE /DUNICODE /DWINAPI_FAMILY=WINAPI_FAMILY_APP /D__WRL_NO_DEFAULT_LIB__ /nologo /Z7 /MP /GS /Gd /Gm- /W3 /WX- /Zc:wchar_t /Zc:inline /Zc:forScope /fp:precise /Oy- /EHsc  /utf-8  /MP /Zm256 /D_DEBUG /MDd /Od /RTC1  /Oi /bigobj -std:c++14   /wd4661 /wd4251 /wd4275 /showIncludes /FoOgreMain\CMakeFiles\OgreMain.dir\src\OgreStaticGeometry.cpp.obj /FdOgreMain\CMakeFiles\OgreMain.dir\ /FS -c C:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\src\OgreStaticGeometry.cpp
C:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\include\OgreWorkQueue.h(194): warning C4244: 'argument': conversion from 'Ogre::uint64' to 'unsigned long', possible loss of data
C:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\include\OgreSceneManager.h(2250): warning C4996: 'Ogre::AnimationStateSet::getAnimationStateIterator': was declared deprecated
C:\v\vcpkg4\buildtrees\ogre\src\v14.0.1-e1473b508d.clean\OgreMain\src\OgreStaticGeometry.cpp(413): warning C4267: '=': conversion from 'size_t' to 'Ogre::uint32', possible loss of data
ninja: build stopped: subcommand failed.
```
</details>**Additional context**

<details><summary>vcpkg.json</summary>

```
{
  "name": "1",
  "version": "1.0",
  "dependencies": [
    {
      "name": "ogre",
      "default-features": false,
      "features": []
    }
  ]
}

```
</details>

--- a/ports/ogre/vcpkg.json
+++ b/ports/ogre/vcpkg.json
@@ -1,9 +1,11 @@
 {
   "name": "ogre",
   "version": "14.0.1",
+  "port-version": 1,
   "description": "3D Object-Oriented Graphics Rendering Engine",
   "homepage": "https://github.com/OGRECave/ogre",
   "license": "MIT",
+  "supports": "!uwp",
   "dependencies": [
     "pugixml",
     {

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -5918,7 +5918,7 @@
     },
     "ogre": {
       "baseline": "14.0.1",
-      "port-version": 0
+      "port-version": 1
     },
     "ogre-next": {
       "baseline": "2.3.1",

--- a/versions/o-/ogre.json
+++ b/versions/o-/ogre.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "ba65238d3998c84df40961d63f7c254fbac7639d",
+      "version": "14.0.1",
+      "port-version": 1
+    },
+    {
       "git-tree": "d244c75813c1f05a64a4a763166fa4f8e3e63eb0",
       "version": "14.0.1",
       "port-version": 0

